editor/questionbankcontroller.js

```diff
@@ -182,6 +195,51 @@ angular.module('createquestionapp', [])
       $scope.searchQuestions();
       $scope.selectedIndex = undefined;
 
+      //Load all plugins
+      $scope.loadPlugins = function(plugins, manifestMedia) {
+        var pluginObj = [];
+        if (!Array.isArray(plugins)) {
+          pluginObj.push(plugins);
+          plugins = pluginObj;
+        }
+        org.ekstep.pluginframework.pluginManager.loadAllPlugins(plugins, manifestMedia, function() {
+          if (typeof PluginManager != 'undefined') {
+            PluginManager.pluginMap = org.ekstep.pluginframework.pluginManager.plugins;
```

Comment:
Why are you writing this function as a scope level function? It should be handled by the plugin-framework itself.